src/main/java/org/threadly/litesockets/tcp/ssl/SSLClient.java

```diff
@@ -246,37 +254,32 @@ public boolean isEncrypted() {
       if(ssle.getHandshakeStatus() == NEED_WRAP) {
         writeForce(ByteBuffer.allocate(0));
       }
-      if(this.ce != null) {
-        ce.getThreadScheduler().schedule(new Runnable() {
+      if(this.seb != null) {
```

Comment:
Possibly, In these cases we do definitly need to throw a timeoutException as failure.  Not sure if that would apply in all watchdog cases.